src/sqlfluff/dialects/dialect_soql.py

```diff
@@ -3,18 +3,16 @@
 https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql.htm
 """
 
+from sqlfluff.core.parser.grammar.anyof import OneOf
+from sqlfluff.core.parser.grammar.sequence import Sequence
```

Comment:
We normally just import these from `sqlfluff.core.parser` in other dialects do we not need to specify each one separately.